packages/syft/tests/syft/core/tensor/adp/phi_tensor_test.py

```diff
@@ -344,6 +344,36 @@ def test_sum(
     assert zeros_tensor.sum().child == 0
 
 
+@pytest.mark.parametrize("kwargs", [{"axis": (1)}])
+def test_mean(
+    reference_data: np.ndarray,
+    upper_bound: np.ndarray,
+    lower_bound: np.ndarray,
+    ishan: DataSubjectArray,
+    kwargs: Dict,
+) -> None:
+    ishan = np.broadcast_to(ishan, reference_data.shape)
+    zeros_tensor = PT(
+        child=reference_data * 0,
+        data_subjects=ishan,
+        max_vals=upper_bound,
+        min_vals=lower_bound,
+    )
+    tensor = PT(
+        child=reference_data,
+        data_subjects=ishan,
+        max_vals=upper_bound,
+        min_vals=lower_bound,
+    )
+
+    tensor_mean = tensor.mean()
+    tensor_mean_axis_1 = tensor.mean(**kwargs)
+
+    assert (tensor_mean.child == reference_data.mean()).all()
+    assert (tensor_mean_axis_1.child == reference_data.mean(**kwargs)).all()
+    assert zeros_tensor.mean().child == 0
```

Comment:
Nicely done! Let's also add a few assert statements to check that the metadata and shape is what we expect it to be, so that we can be sure that DP noise is added correctly :)

Reply:
I added a few more `assert` checks, but let me know if there are other props I should be testing for.

Reply:
I'd also check that the min_vals and max_vals have the same shape as the child- but that's all :)

Reply:
Added!